pkg/descheduler/strategies/duplicates.go

```
@@ -85,9 +85,6 @@ func FindDuplicatePods(pods []*v1.Pod) DuplicatePodsMap {
 		if err != nil || sr == nil {
 			continue
 		}
```

remove this check also.

---

I need sr in the next line, so I had to include that.

---

but the check is not needed i think, because all pods with sr==nil or err != nil are already checked in ListEvictablePodsOnNode so no pod at this stage should have sr==nil or err != nil.

---

The reason is - I have to ignore err which is second argument and take sr only in order to remove this check. Will make the change now.